src/install.rst

```diff
@@ -17,11 +17,12 @@ When completed, you'll be ready to run Nextstrain :term:`workflows <workflow>`.
 Installation steps
 ==================
 
-Steps vary by runtime option (Docker, native) and operating system (macOS, Windows, WSL on Windows, Linux).
+Steps vary by runtime option (Docker, ambient) and operating system (macOS, Windows, WSL on Windows, Linux).
```

Comment:
Re-ordered the commits so that 5bc7a5d comes before 18a6a0f, since 18a6a0f references the new name.

Reply:
Thanks!